packages/ui-components/src/components/Chart/Table/getChartTableData.js

```diff
@@ -22,13 +22,19 @@ const FirstColumnCell = styled.span`
   text-align: left;
 `;
 
-const makeFirstColumn = (header, accessor, sortRows) => ({
-  Header: header,
-  accessor,
-  sortType: sortRows,
-  // eslint-disable-next-line react/prop-types
-  Cell: ({ value }) => <FirstColumnCell>{String(value)}</FirstColumnCell>,
-});
+const makeFirstColumn = (header, accessor, sortRows) => {
+  const firstColumn = {
+    Header: header,
+    accessor,
+    // eslint-disable-next-line react/prop-types
+    Cell: ({ value }) => <FirstColumnCell>{String(value)}</FirstColumnCell>,
+  };
+  if (sortRows) {
+    firstColumn.sortType = sortRows;
+  }
```

Comment:
This fixes a bug where having `sortType` set to `undefined` broke sorting for non-date columns